### PR TITLE
DOCSP-14310: Android Kotlin Tutorial SDK Version Update

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- https://jira.mongodb.org/browse/DOCSP-14310
   - ended up holding off on any major refactoring changes for simplicity (splitting out models creates far too many files and lengthens the tutorial significantly)

### Staged Changes (Requires MongoDB Corp SSO)

- [Tutorial/Kotlin](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-14310/tutorial/android-kotlin/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: tutorial/kotlin-android/app/src/main/java/com/mongodb/tasktracker/TaskTracker.kt [hunk 1]
```diff
@@ -31,7 +31,9 @@ class TaskTracker : Application() {
             AppConfiguration.Builder(BuildConfig.MONGODB_REALM_APP_ID)
             .build())
         // :state-end: :state-uncomment-start: start
-        //// TODO: Initialize the Realm SDK and create the App object we will use to communicate with the Realm backend.
+        //// Initialize the Realm SDK
+        //Realm.init(this)
```
Comment (cbush) [inline tutorial/kotlin-android/app/src/main/java/com/mongodb/tasktracker/TaskTracker.kt:35]: Why not move both `Realm.init(this)`s out of the state blocks?

**Review by cbush — APPROVED** [review]
LGTM

File: tutorial/kotlin-android/app/src/main/java/com/mongodb/tasktracker/TaskTracker.kt (hunk 1 above)
Comment (jenniferpeshansky) [inline tutorial/kotlin-android/app/src/main/java/com/mongodb/tasktracker/TaskTracker.kt]: is this meant to be commented out?

File: source/includes/steps-tutorial-android-kotlin.yaml [hunk 2]
```diff
@@ -451,21 +452,21 @@ content: |
...
-    button and entering "*first@example.com*"
-  - Log out and log in as *first@example.com*
-  - You should see two projects in the projects list, one of them labeled "*second@example.com*'s project"
-  - Navigate to *second@example.com*'s project
-  - Collaborate by adding, updating, and removing some new tasks
+  1. Create a user with email *first@example.com*.
+  #. Explore the app, then log out or launch a second instance of the app
+     on another virtual device.
+  #. Create another user with email *second@example.com*.
+  #.  Navigate to *second@example.com*'s project.
```
Comment (jenniferpeshansky) [inline source/includes/steps-tutorial-android-kotlin.yaml:471]: is there too much space here before "navigate"?

**Review by jenniferpeshansky — APPROVED** [review]
LGTM mod a few questions

**Review by jenniferpeshansky — APPROVED** [review]
LGTM mod a few questions

**Pull request merged**